assets/css/fonts.css

```diff
@@ -3,4 +3,25 @@ h1, h2, h3, h4, h5, h6 {
 }
 p {
   @apply text-lg;
+}
+p:not(:last-child) {
+  @apply mb-4;
+}
```

Comment:
Let's scope the styles like this to their containers to save having to reset them. The `.prose` class should format things nicely wherever we need it